mypaint-brush.c

```diff
@@ -714,6 +714,8 @@ smallest_angular_difference(float a, float b)
         b = self->states[MYPAINT_BRUSH_STATE_LAST_GETCOLOR_B];
         a = self->states[MYPAINT_BRUSH_STATE_LAST_GETCOLOR_A];
       }
+      //convert to RYB-- we want to mix the smudge with the new getcolor in RYB space
+      rgb_to_ryb_float (&r, &g, &b);
```

Comment:
That makes a lot of sense and avoids messing with MyPaint preferences...  I don't know why someone would want to crossfade the mix but that's never a good reason not to allow it ;-).  Maybe I'll wrap both methods up inside if statements so they only execute if necessary and then do yet another crossfade at the end...

so something like 
if MixSetting <1 {
 do rgb mix }
if MixSetting >0 {
 do ryb mix }
At this point should I just do a crossfade no matter what?  Rr avoid redundant crossfade like
if MixSetting = 0 {
  use rgb_mix }
 else if MixSetting = 1 {
  use ryb_Mix }
 else {
crossfade rgb w/ ryb }